lib/models/service_model.dart

```diff
@@ -1,29 +1,48 @@
 import 'package:flutter/material.dart';
 
-enum ServiceGroup { wallet, trade, bets }
+enum ServiceGroup { wallet, trade, bets, invest }
 
 extension ServiceGroupExtension on ServiceGroup {
   static const labels = {
     ServiceGroup.wallet: "Wallet",
     ServiceGroup.trade: "Trading",
-    ServiceGroup.bets: "Bets"
+    ServiceGroup.bets: "Bets",
+    ServiceGroup.invest: "Stacking"
   };
   static const icons = {
     ServiceGroup.wallet: Icons.wallet,
     ServiceGroup.trade: Icons.query_stats,
-    ServiceGroup.bets: Icons.casino
+    ServiceGroup.bets: Icons.casino,
+    ServiceGroup.invest: Icons.table_rows
   };
 
   String get label => labels[this]!;
   IconData get icon => icons[this]!;
 }
 
-enum Service { cfd, sportsbet }
+enum Service { cfd, sportsbet, exchange, savings }
 
 extension ServiceExtension on Service {
-  static const labels = {Service.cfd: "CFD Trading", Service.sportsbet: "Sports Bets"};
-  static const icons = {Service.cfd: Icons.insights, Service.sportsbet: Icons.sports_football};
+  static const labels = {
+    Service.cfd: "CFD Trading",
+    Service.sportsbet: "Sports Bets",
+    Service.exchange: "Taro Exchange",
+    Service.savings: "Saving"
+  };
+  static const shortLabels = {
+    Service.cfd: "CFDs",
+    Service.sportsbet: "Bets",
```

Comment:
Nice idea :) 

Is it possible to disable them or show them in a lighter color do indicate they are disabled?

Reply:
How about adding "coming soon" screens instead? - then the button would be enabled but there's only a screen that says coming soon (potentially with some more words on what is planned...?) 
I imagine that could be interesting if anybody joins the demo.